x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_form.tsx

```diff
@@ -297,6 +302,27 @@ export const AgentPolicyForm: React.FunctionComponent<Props> = ({
           }}
         />
       </EuiDescribedFormGroup>
+      <EuiDescribedFormGroup
+        title={<h4>{unenrollmentTimeoutText}</h4>}
+        description={
+          <FormattedMessage
+            id="xpack.fleet.agentPolicyForm.unenrollmentTimeoutDescription"
+            defaultMessage="An optional timeout in seconds. If provided, an agent will automatically unenroll after being gone for this period of time."
+          />
+        }
+      >
+        <EuiFormRow fullWidth>
+          <EuiFieldNumber
+            fullWidth
+            value={agentPolicy.unenroll_timeout}
+            min={1}
```

Comment:
set a minimum of `1` to prevent `0` (as we use `null`) or any negative numbers.

I'm not sure if there should be a `max`. cc: @nchaulet @mostlyjason @blakerouse

Reply:
Not sure neither unless there is a technical reason to have a max on the fleet server side.